### DISCUSSION TITLE
Triton Management Service is discontinued?

### DISCUSSION THREAD

**Discussion by chenzhiwei** [triton-inference-server/server#7401] [Q&A] [answered]
Hello, I just found TMS will be EOL on July 1st, any reasons why it is discontinued and are there any replacement?

TMS is a good product and seems it can bring benefits to our deployment.

/cc @dyastremsky

**Comment by the-david-oy** [comment]
CC: @nnshah1 @dzier @whoisj

**Comment by MrBoJo84** [comment, marked as answer]
Hello @chenzhiwei, please reach out to your NVIDIA representative for more information. If you don't know who that is, please open a support ticket via NVIDIA Enterprise Support.
